ArraysHomework/ArraysHomework.cs

```diff
@@ -55,7 +63,8 @@ public void Arrays()
 
             // Swap the last and first value of this array.
             string[] swapMyValues = { "first", "middle", "last" };
-            
+            swapMyValues[0] = "last";
```

Comment:
This technically does the job, but only because you know what the first and last value are (because you can see where the array is initialized).
How would you do this if you didn't know the values (for example, if they came from user input)?